tests/main/snapshot-cross-revno/task.yaml

```diff
@@ -19,7 +19,7 @@ execute: |
     fi
 
     for u in root test; do
-        su $login_option -c "
+        su "$login_option" -c "
```

Comment:
this is causing google:arch-linux-64:tests/main/snapshot-cross-revno to fail

Reply:
Ouch... solved by ignoring warning